<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZELNLicenseManagerDialog</class>
 <widget class="QWidget" name="ZELNLicenseManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zinek License Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ZELNLauncher.qrc">
    <normaloff>:/ZELNLauncher/Resources/ZinekEngine.ico</normaloff>:/ZELNLauncher/Resources/ZinekEngine.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Application Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tblLicenses">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>GUID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Valid</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Licensee</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Serial Key</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Product</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Version</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Edition</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Scope</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Enabled</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnViewEdit">
         <property name="text">
          <string>View/Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnImport">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExport">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tblLicenses</tabstop>
  <tabstop>btnViewEdit</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnExport</tabstop>
 </tabstops>
 <resources>
  <include location="ZELNLauncher.qrc"/>
 </resources>
 <connections/>
</ui>
